app.py

```diff
@@ -105,6 +114,12 @@ def get_light():
     response['light'] = randData
     return response
 
+def generate_random_values(min, max):
+    values = []
+    for i in range(0,10):
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)